mne/io/eeglab/eeglab.py

```diff
@@ -525,8 +525,13 @@ def _read_eeglab_events(eeg, event_id=None, event_id_func='strip_to_integer'):
     if event_id is None:
         event_id = dict()
 
-    types = [event.type for event in eeg.event]
-    latencies = [event.latency for event in eeg.event]
+    try:
+        types = [event.type for event in eeg.event]
+        latencies = [event.latency for event in eeg.event]
+    except TypeError:
```

Comment:
Something like

`types = [event.type for event in eeg.event] if len(eeg.event) > 1 else [eeg.event.type]`

or is that 'too clever' ..?